inesita.gemspec

```diff
@@ -15,14 +15,14 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.require_paths = ['lib']
 
-  s.add_dependency 'opal', '>= 0.9', '< 0.11'
+  s.add_dependency 'opal-sprockets', '~> 0.4.0.0.11.0.rc1.3.1.beta'
```

Comment:
@elia Thanks, I was wondering why `opal-sprockets` version is weird. Now I know. Fixed.